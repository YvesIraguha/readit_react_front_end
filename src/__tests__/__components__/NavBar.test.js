import React from "react";
import { shallow } from "enzyme";
import Navbar from "../../components/common/NavBar";

describe("Navigation bar", () => {
  test("should render the navigation bar", () => {
    const wrapper = shallow(<Navbar />);
    expect(wrapper).toMatchSnapshot();
  });
});
