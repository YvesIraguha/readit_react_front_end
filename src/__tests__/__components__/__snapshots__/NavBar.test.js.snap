// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation bar should render the navigation bar 1`] = `
<section
  className="navbar"
>
  <div
    className="logo"
  >
    <img
      alt="Logo"
      className="icon"
      src="test-file-stub"
    />
  </div>
  <div
    className="profile"
  >
    <div
      className="avatar"
    >
      <img
        alt="Profile"
        className="icon"
        src="test-file-stub"
      />
    </div>
    <div
      className="username"
    >
      <p>
        Joe Doe
      </p>
    </div>
  </div>
</section>
`;
